### ISSUE TITLE
layout diagram in README not showing up

### ISSUE THREAD

**Issue by ryancoe** [SNL-WaterPower/siweed#95]
It looks like a bad link (or is it just me?)

**Comment by nickross4444** [comment]
I've updated it for now, but we will need to do so again once the hardwareX branch is merged into develop, and maybe again for master, since the link needs to point to the right branch.

**Comment by ryancoe** [comment]
this is probably the sort of change you can make directly on the master branch
